### ISSUE TITLE
Extensor carpi ulnaris tendon subluxation / instability

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10753]
**Preferred term label:**
I don't know if there should be one or 2 terms in HPO, but the instability term is one detected on ultrasound without the subluxation being observable clinically

Extensor carpi ulnaris tendon instability
Extensor carpi ulnaris tendon subluxation

**Synonyms**
ECU

**Definition (free text, please give PubMed ID)**
Instability: Dynamic ultrasound of the wrist revealing ECU tendon subluxating out of the distal ulnar groove
Subluxation: Translation of the ECU tendon out of its groove in the ulnar head with wrist flexion and ulnar deviation

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Extensor carpi ulnaris (ECU) tendon subluxation
Extensor carpi ulnaris (ECU) tendonitis

**Your nano-attribution (ORCID)**
